great_expectations/expectations/core/expect_table_row_count_to_equal_other_table.py

```diff
@@ -40,6 +40,9 @@ class ExpectTableRowCountToEqualOtherTable(BatchExpectation):
     expect_table_row_count_to_equal_other_table is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
+    BatchExpectations are one of the most common types of Expectation.
+    They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
+
     Args:
         other_table_name (str): \
             The name of the other table.
```

Comment:
Specifying a table name is somewhat unclear with respect to Data Sources and Data Assets. How does GX query for the other table row count relative to the table that the expectation is being run on? For example, is it assumed to be in the same db, same schema?